b-real-time-htb-validator.js

```diff
@@ -42,7 +42,11 @@ var partnerValidator = function (configs) {
                         type: 'object',
                         properties: {
                             placementId: {
-                                type: 'string',
+                                type: 'number',
+                                minLength: 1
+                            },
+                            sizes: {
+                                type: 'array',
```

Comment:
Hi BRealTime Team,
Can you please add schema validation for the items in the array as well? Please refer to this doc: https://github.com/Atinux/schema-inspector#v_items for more information.